src/main.rs

```diff
@@ -159,6 +159,45 @@ fn main() {
     }
 }
 
+#[cfg(not(feature = "with_server"))]
+fn push_to_server(_: &str, _: usize, _: usize) -> Result<(), &'static str> {
```

Comment:
I should have made a separate commit for this but a47611e2d452525ff60714a179b3e5f43c103ddb also tweaked the signature for the non-default version of push_to_server.

Reply:
Oops. Good catch.